Xamarin.Forms.Platform.Android/AppCompat/MasterDetailContainer.cs

```diff
@@ -106,6 +107,37 @@ protected override void AddChildView(VisualElement childView)
 			}
 		}
 
+		protected override void Dispose(bool disposing)
+		{
+			if (_disposed)
+			{
+				return;
+			}
+
+			_disposed = true;
```

Comment:
When you call `Dispose(false)`, it sets `_disposed` to `true` but does `NOT` actually dispose (the if block doesn't get executed). When you call `Dispose(true)` later on, it assumes the object is already disposed, so it returns. `MasterDetailPageRenderer.cs` and `NavigationPageRenderer.cs` have the correct implementation.